scala-package/examples/src/test/scala/org/apache/mxnetexamples/infer/profiler/ProfilerTest.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.mxnetexamples.profiler
+
+import org.scalatest.{BeforeAndAfterAll, FunSuite}
+import org.slf4j.LoggerFactory
+import java.io.File
+
+import org.apache.mxnet.Profiler
+import org.apache.mxnet.Context
+
+/**
+  * Integration test for imageClassifier example.
+  * This will run as a part of "make scalatest"
+  */
+class ProfilerTest extends FunSuite with BeforeAndAfterAll {
```

Comment:
better to move tests to `test` package,  use `before{` for `Profiler.profilerSetState("run")`, `after{` for `Profiler.profilerSetState("stop")`, and move each `testXXX` in different `test{` scope: http://www.scalatest.org/getting_started_with_fun_suite